source/docs/appendices/migration-guides.md

```diff
@@ -9,6 +9,11 @@ Our goal is to make migrations as smooth as possible. This is why we try to make
 
 ## `2.5.0` -> `2.6.0`
 
+### Minimum Node.js version
+
+Node.js 10.x reaches the end of support in the end of April 2021. As a result,
+the minimum Node.js required version has been updated to the version 12.22.1.
```

Comment:
It could be good to add a note that we recommend users to update their servers to 14.x directly (since this LTS is currently supported by FC). That way, when developers will check their node versions they could upgrade to an actively maintained node version!

Reply:
done in 907eaeb